llm_studio/src/utils/modeling_utils.py

```diff
@@ -35,6 +36,30 @@ def unwrap_model(model: torch.nn.Module):
     return model
 
 
+def check_disk_space(model: torch.nn.Module, path: str):
```

Comment:
@haqishen is this costing in terms of runtime?

did you benchmark it also on larger backbones?

Reply:
it takes almost no time for this function, like `0:00:00.007159` or so for 7b model.